po/pt_BR.po

```diff
@@ -0,0 +1,453 @@
+msgstr "Desemparelhar"
+
+#: lib/solaar/ui/config_panel.py:98
+msgid "Working"
+msgstr "Funcionando"
+
+#: lib/solaar/ui/config_panel.py:101
+msgid "Read/write operation failed."
+msgstr "Operação de leitura/escrita falhou."
+
+#: lib/solaar/ui/notify.py:120
+msgid "connected"
+msgstr "conectado"
+
+#: lib/solaar/ui/notify.py:122 lib/solaar/ui/tray.py:290
+#: lib/solaar/ui/tray.py:295 lib/solaar/ui/window.py:653
+msgid "offline"
+msgstr "offline"
+
+#: lib/solaar/ui/pair_window.py:133
+msgid "Pairing failed"
+msgstr "Pareamento falhou"
+
+#: lib/solaar/ui/pair_window.py:135
+msgid "Make sure your device is within range, and has a decent battery charge."
+msgstr "Certifique-se de que o seu dispositivo esteja dentro do alcance e, que possui bateria com carga aceitável."
+
+#: lib/solaar/ui/pair_window.py:137
+msgid "A new device was detected, but it is not compatible with this receiver."
+msgstr "Um novo dispositivo foi detectado mas, não é compatível com este receptor."
+
+#: lib/solaar/ui/pair_window.py:139
+#, python-format
+msgid "The receiver only supports %d paired device(s)."
+msgstr "O receptor só suporta %d dispositivo(s) emparelhado(s)"
+
+#: lib/solaar/ui/pair_window.py:141
+msgid "No further details are available about the error."
+msgstr "Sem maiores detalhes sobre o erro."
+
+#: lib/solaar/ui/pair_window.py:155
+msgid "Found a new device"
+msgstr "Novo dispositivo encontrado"
+
+#: lib/solaar/ui/pair_window.py:180
+msgid "The wireless link is not encrypted"
+msgstr "O link sem fio não é criptografado"
+
+#: lib/solaar/ui/pair_window.py:197
+msgid "pair new device"
+msgstr "emparelhar novo dispositivo"
+
+#: lib/solaar/ui/pair_window.py:205
+msgid "Turn on the device you want to pair."
+msgstr "Ligue o dispositivo que você quer emparelhar."
+
+#: lib/solaar/ui/pair_window.py:206
+msgid ""
+"If the device is already turned on,\n"
+"turn if off and on again."
+msgstr ""
+"Se o dispositivo já encontra-se ligado,\n"
+"desligue-o e ligue-o novamente."
+
+#: lib/solaar/ui/tray.py:55
+msgid "No Logitech receiver found"
+msgstr "Nenhum receptor da Logitech encontrado"
+
+#: lib/solaar/ui/tray.py:62
+msgid "Quit"
+msgstr "Sair"
+
+#: lib/solaar/ui/tray.py:274
+msgid "no receiver"
+msgstr "nenhum receptor"
+
+#: lib/solaar/ui/tray.py:293
+msgid "no status"
+msgstr "sem status"
+
+#: lib/solaar/ui/window.py:58
+msgid "The wireless link between this device and its receiver is encrypted."
+msgstr "O link sem fio entre o dispositivo e seu receptor é criptografado."
+
+#: lib/solaar/ui/window.py:59
+msgid ""
+"The wireless link between this device and its receiver is not encrypted.\n"
+"\n"
+"For pointing devices (mice, trackballs, trackpads), this is a minor security "
+"issue.\n"
+"\n"
+"It is, however, a major security issue for text-input devices (keyboards, "
+"numpads),\n"
+"because typed text can be sniffed inconspicuously by 3rd parties within "
+"range."
+msgstr ""
+"O link sem fio entre o dispositivo e seu receptor é criptografado.\n"
```

Comment:
"[...] **não** é criptografado[...]" -> wrong translation, English says "not encrypted" and Portuguese is saying "encrypted"